src/test/java/com/puppycrawl/tools/checkstyle/grammar/javadoc/GeneratedJavadocTokenTypesTest.java

```diff
@@ -146,6 +150,17 @@ public void testTokenNumbers() {
         assertEquals(89, JavadocParser.ATTR_VALUE, MSG);
         assertEquals(90, JavadocParser.Char12, MSG);
         assertEquals(91, JavadocParser.HTML_COMMENT_END, MSG);
+
```

Comment:
Please move `assertEquals(101, ..` to most bottom of aseert list.
It confused me, why count is 101, while last is 91.
It would be awesome to order last asserts.

Reply:
Done.